packages/web-app-files/tests/unit/views/Trashbin.spec.js

```diff
@@ -0,0 +1,206 @@
+import { mount } from '@vue/test-utils'
+import Trashbin from '@files/src/views/Trashbin.vue'
+import { getStore, localVue, createFile } from '@files/tests/unit/views/views.setup.js'
+
+const stubs = {
+  'list-loader': true,
+  'no-content-message': true,
+  'resource-table': true,
+  'context-actions': true,
+  pagination: true,
+  'list-info': true
+}
+
+const listLoaderStub = 'list-loader-stub'
+const noContentStub = 'no-content-message-stub'
+const filesTableStub = 'resource-table-stub'
+const contextActionsStub = 'context-actions-stub'
+const listInfoStub = 'list-info-stub'
+const paginationStub = 'pagination-stub'
+
+describe('Trashbin View', () => {
+  it('should show the list loader when the view is still loading', () => {
+    const wrapper = getMountedWrapper({ loading: true })
+
+    expect(wrapper.find(listLoaderStub).exists()).toBeTruthy()
+    expect(wrapper.find(noContentStub).exists()).toBeFalsy()
+    expect(wrapper.find(filesTableStub).exists()).toBeFalsy()
+  })
+
+  describe('when the view is not loading anymore', () => {
+    it('should show the no content message component if the paginated resources is empty', () => {
+      const wrapper = getMountedWrapper()
+
+      expect(wrapper.find(listLoaderStub).exists()).toBeFalsy()
+      expect(wrapper.find(filesTableStub).exists()).toBeFalsy()
+      expect(wrapper.find(noContentStub).exists()).toBeTruthy()
+    })
+
+    describe('when length of the paginated resources is greater than zero', () => {
+      const resourceList = [
+        createFile({ id: '1234' }),
```

Comment:
What speaks against using the fixtures for deleted files here? In my opinion there's some value in having the data structure of those present in the Trashbin View unit test :)

https://github.com/owncloud/web/blob/0d22451b8a439a0819af726695e857d49dd1dc61/__fixtures__/deletedFiles.js

Reply:
These data are not appropriate for the resource table component:

> Resources to be displayed in the table.
> Required fields:
> - name: The name of the resource containing the file extension in case of a file
> - path: The full path of the resource
> - type: The type of the resource. Can be `file` or `folder`
> Optional fields:
> - thumbnail
> - size: The size of the resource
> - modificationDate: The date of the last modification of the resource
> - shareDate: The date when the share was created
> - deletionDate: The date when the resource has been deleted
> - status: The status of the share. Contains also actions to accept/decline the share
> - opensInNewWindow: Open the link in a new window

_above information are form the prop docs in ResourceTable.vue_